pkg/minikube/problem/err_map.go

```diff
@@ -119,6 +125,12 @@ var vmProblems = map[string]match{
 		URL:    "http://mikko.repolainen.fi/documents/virtualization-with-kvm",
 		Issues: []int{2991},
 	},
+	"OLD_KVM_VERSION": {
+		Regexp: re(`Too old KVM version found`),
```

Comment:
Which message is this regexp matching? Do you have an example you a show of where this message is displayed to the user?